test/client_test.rb

```diff
@@ -181,11 +181,15 @@ class ClientTest < Minitest::Test
       client = Vimaly::Client.new('company_id', user_credentials: { username: 'username', password: 'password' })
       bins = client.bins
 
-      assert_equal 2, bins.size
+      assert_equal 4, bins.size # API called twice now (2 pages)
```

Comment:
LGTM, so our Mocked API is returning `[Alpha, Beta]` for both requests?  Eg. `page=2` is still returning `[Alpha, Beta]`?   Clearly a fake mock, but works for me.

Reply:
Something a bit more?  Out-of-context I think it's confusing.

```suggestion
      # API called twice now (2x pages returning a mocked result of 2 bins)
      assert_equal 4, bins.size
```